modules/javafx.graphics/src/main/java/javafx/concurrent/Service.java

```diff
@@ -741,74 +741,31 @@ private EventHelper getEventHelper() {
         return eventHelper;
     }
 
-    /**
-     * Registers an event handler to this task. Any event filters are first
-     * processed, then the specified onFoo event handlers, and finally any
-     * event handlers registered by this method. As with other events
-     * in the scene graph, if an event is consumed, it will not continue
-     * dispatching.
-     *
-     * @param <T> the specific event class of the handler
-     * @param eventType the type of the events to receive by the handler
-     * @param eventHandler the handler to register
-     * @throws NullPointerException if the event type or handler is null
-     * @since JavaFX 2.1
-     */
+    @Override
```

Comment:
By removing the docs, you lost the `@since` for this method, so you will need to restore it. You might consider also adding `{@inheritDoc}` to avoid copying all of the description